monodbassignment/admins.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
You shouldn't have multiple collection for each role. 
There should be just one collection for users. 
It should have a role field which determines the role of that user.